applications/adaptive-learning/configs/config_engine.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/python3
+import configparser
+import threading
+
+
+class ConfigEngine:
```

Comment:
Later you can use the latest version of config parser from the new repo containing the smart-social-distancing code.